app/views/catalog/index.html.erb

```diff
@@ -31,3 +34,28 @@
     <%= paginate @page_eq_models_by_category %>
   <% end %>
 </div>
+
+<script>
+$(document).ready(function() {
+  $('#toggle_unavailable').width(
+    Math.max.apply(
+      Math,
+      $('#toggle_unavailable').map(function(){
+        return $(this).width();
+      }).get()
+    )
+  );
+  $("#toggle_unavailable").click(function() {
+    if ($("#toggle_unavailable").text() == "Hide unavailable") {
+      $("#toggle_unavailable").text("Show unavailable");
+    } else {
+      $("#toggle_unavailable").text("Hide unavailable");
+    }
+    $(".col-md-4").each(function() {
```

Comment:
While this works, could we use class/div tags and attach to those instead of the (".col-md-4") and (".col-md-3") css tags? (Ex. in the future we might make something else width 4, and wonder why it suddenly keeps disappearing) :)